File: kotlin/day01.kt
import java.io.File

fun readInput(day: Int): List<String> = File("../inputs/day%02d.txt".format(day)).readLines()

fun part1(data: List<String>): Any? {
    return null
}

fun part2(data: List<String>): Any? {
    return null
}

fun main() {
    val data = readInput(1)
    println("Part 1: ${part1(data)}")
    println("Part 2: ${part2(data)}")
}

File: go/day01.go
package main

import (
	"fmt"
	"log"
)

func part1(data []string) any {
	return nil
}

func part2(data []string) any {
	return nil
}

func main() {
	data, err := ReadInput(1)
	if err != nil {
		log.Fatal(err)
	}

	fmt.Println("Part 1:", part1(data))
	fmt.Println("Part 2:", part2(data))
}
